arangod/Agency/MoveShard.cpp

```diff
@@ -1551,6 +1551,10 @@ void MoveShard::addUndoMoveShard(Builder& ops, Builder const& job) const {
     ops.add("timeStamp", VPackValue(now));
     ops.add("removeIfNotStartedBy", VPackValue(deadline));
     ops.add("rebootId", VPackValue(rebootId.value()));
-    ops.add("moveShard", job.slice());
+    ops.add("type", "moveShard");
+    ops.add("database", _database);
+    ops.add("collection", _collection);
+    ops.add("fromServer", _from);
+    ops.add("toServer", _to);
```

Comment:
This change can break existing cluster that contain undo operations. We should not interfere with existing implementations.